readme.md

```diff
@@ -44,6 +44,12 @@ You have to explicitly ignore the parent directories too:
 del.sync(['public/assets/**', '!public/assets', '!public/assets/goat.png']);
 ```
 
+To delete all sub-directories inside `public/`, you can do:
```

Comment:
This doesn't look related to the above. It should be somewhere else. It's also not clear from the text whether this deletes all sub-directories *and* files directly under `public/` or just it's sub-directories.

Reply:
where should I put it? it only deletes all sub-directories inside `public/` (no files are deleted)

Reply:
In a `## Tips` section.

> it only deletes all sub-directories inside public/ (no files are deleted)

That should be made clear in the text.